bouncer/src/repo/manipulator/modelconvertor/import/repo_model_import_synchro.cpp

```diff
@@ -640,32 +679,37 @@ void SynchroModelImport::generateTaskInformation(
 	}
 
 	scene->addSequenceTasks(taskBSONs, generateTaskCache(rootTasks, taskToChildren));
+
+	return firstTS;
 }
 
 repo::core::model::RepoScene* SynchroModelImport::generateRepoScene(uint8_t &errMsg) {
 	std::unordered_map<std::string, std::vector<repo::lib::RepoUUID>> resourceIDsToSharedIDs;
+	std::unordered_map<std::string, repo::lib::RepoUUID> resourceIDsToRootTransID;
 
 	repo::core::model::RepoScene* scene = nullptr;
 	try {
 		repoInfo << "Constructing scene...";
-		scene = constructScene(resourceIDsToSharedIDs);
+		scene = constructScene(resourceIDsToSharedIDs, resourceIDsToRootTransID);
 
 		repoInfo << "Getting tasks... ";
 
-		generateTaskInformation(reader->getTasks(), resourceIDsToSharedIDs, scene);
+		auto firstFrame = generateTaskInformation(reader->getTasks(), resourceIDsToSharedIDs, scene);
 
 		repoInfo << "Getting animations... ";
-		auto animInfo = reader->getAnimation();
-
-		auto animation = animInfo.first;
+		auto animation = reader->getAnimation();
```

Comment:
I'm getting `'struct pair<Animation, unordered_map<string, bool>>' has no member named '...'` errors on lines, 712, 737, 767, 770, 780, 800 for things like `name`, `lastFrameVisibility`, etc.